metsim/methods/mtclim.py

```diff
@@ -26,74 +21,61 @@ def run(forcing: pd.DataFrame, disagg=True):
     lat_idx = forcing.index.names.index('lat')
     time_idx = forcing.index.names.index('time')
     sg = solar_geom(forcing['elev'][0], forcing.index.levels[lat_idx][0])
+    sg = {'tiny_rad_fract' : sg[0],
```

Comment:
Yup, it's the same principle around variable swapping (eg `a, b = b, a`)  In Python the right hand side is evaluated before the assignment is made.